integrations/access/accessrequest/app.go

```diff
@@ -185,6 +223,38 @@ func (a *App) onWatcherEvent(ctx context.Context, event types.Event) error {
 	}
 }
 
+func (a *App) handleAccessMonitoringRule(ctx context.Context, event types.Event) error {
+	if kind := event.Resource.GetKind(); kind != types.KindAccessMonitoringRule {
+		return trace.BadParameter("expected %s resource kind, got %s", types.KindAccessMonitoringRule, kind)
+	}
+
+	e, ok := event.Resource.(types.Resource153Unwrapper)
+	if !ok {
+		return trace.BadParameter("expected AccessMonitoringRule resource type, got %T", event.Resource)
```

Comment:
Technically you're expecting a `Resource153Unwrapper`, not `AccessMonitoringRule` here.

That said, this error message isn't going to make a lot of sense to a user.